src/lib.rs

```diff
@@ -292,3 +297,39 @@ pub fn log_file<P: AsRef<Path>>(path: P) -> io::Result<File> {
         .append(true)
         .open(path)
 }
+
+/// Convenience method for opening a re-openable log file with common options.
+///
+/// The file opening is equivalent to:
+///
+/// ```no_run
+/// std::fs::OpenOptions::new()
+///     .write(true)
+///     .create(true)
+///     .append(true)
+///     .open("filename")
+/// # ;
+/// ```
+///
+/// See [`OpenOptions`] for more information.
+///
+/// [`OpenOptions`]: https://doc.rust-lang.org/std/fs/struct.OpenOptions.html
+#[cfg(all(not(windows), feature = "reopen-03"))]
+#[inline]
+pub fn log_reopen(path: &PathBuf, signal: Option<libc::c_int>) -> io::Result<reopen::Reopen<File>> {
```

Comment:
I think taking `&Path` would be strictly more general than `&PathBuf`, here. `&PathBuf` can coerce into `&Path`, but not the other way around.

Reply:
I think I tried &Path, but that did not go down as I thought it would for some reason. I'm a rust newbie, so I guess I missed something. I'll look into that and get back to you.

Reply:
OK, seems to work :)